src/commands/help-command.js

```diff
@@ -0,0 +1,33 @@
+import { MessageEmbed } from 'discord.js';
+
+export class HelpCommand {
+    static getCommand() {
+        return 'help';
+    }
+
+    static create(message, logger) {
+        return new HelpCommand(message, logger);
```

Comment:
Every command should have a static constructor that should accept a Discord message object and a logger.